### ISSUE TITLE
Feature dump of minimal APIs

### ISSUE THREAD

**Issue by davidfowl** [Minimal-APIs/minimal-apis.github.io#57]
https://gist.github.com/davidfowl/ff1addd02d239d2d26f4648a06158727

**Comment by davidfowl** [comment]
@LadyNaggaga I made a few more changes and added a TOC:

https://gist.github.com/davidfowl/ff1addd02d239d2d26f4648a06158727#minimal-apis-at-a-glance

**Comment by LadyNaggaga** [comment]
I will add the content update 👍🏿 . We may not need a TOC because it's auto populated in the side menu.

![image](https://user-images.githubusercontent.com/2546640/133000529-e5787c40-90ed-4209-8c96-fadb4eed2299.png)

**Comment by LadyNaggaga** [comment]
> This listens to port http://localhost:5000 and https://localhost:5001 by default.

Should this section be updated as well since, when some creates an application today, they will get a random port ?

**Comment by davidfowl** [comment]
Yes I'll remove that part

**Comment by davidfowl** [comment]
> I will add the content update 👍🏿 . We may not need a TOC because it's auto populated in the side menu.

I would put WebApplication and WebApplicationBuilder under Application.

**Comment by LadyNaggaga** [comment]
https://gist.github.com/davidfowl/ff1addd02d239d2d26f4648a06158727#differences-with-aspnet-core-mvc

I like this section it should be a detailed blog post.  How do you feel about title being Minimal APIs Vs MVC?  The thing about this section is we may need to create section the highlights when you should use minimal or MVC.  Which is something we are trying to avoid; I thought.  Thoughts?

> ## Differences with ASP.NET Core MVC
> 
> - No support for filters. i.e. `IAsyncAuthorizationFilter`, `IAsyncActionFilter`, `IAsyncExceptionFilter`, `IAsyncResultFilter`, `IAsyncResourceFilter`
> - No support for model binding. i.e. `IModelBinderProvider`, `IModelBinder`. Support can be added with a custom binding shim.
>   - No support for binding from forms. This includes binding `IFormFile` (this will be added in the future).
> - No built-in support for validation. i.e. `IModelValidator`
> - No support for application parts or the application model. There's no way to apply or build your own conventions.
> - No built-in view rendering support. We recommend using Razor Pages for rendering views.
> - No support for [JsonPatch](https://www.nuget.org/packages/Microsoft.AspNetCore.JsonPatch/)
> - No support for [OData](https://www.nuget.org/packages/Microsoft.AspNetCore.OData/)
> - No support for [ApiVersioning](https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Versioning/), see [this issue](https://github.com/dotnet/aspnet-api-versioning/issues/751) for more details.

**Comment by davidfowl** [comment]
I don't think this should be on this site but it should go into https://docs.asp.net. I also don't think it should be a blog post.

**Comment by LadyNaggaga** [comment]
Fair enough.  My hesitation on this section is it's a lot of information that may take someone down a Google rabbit hole.

**Comment by davidfowl** [comment]
> Fair enough. My hesitation on this section is it's a lot of information that may take someone down a Google rabbit hole.

Right, that's why it should be left out for this site and should only appear on docs.asp.net. That's where the existing .NET devs will go to find that information.

**Comment by LadyNaggaga** [comment]
Cool it looks like we are on the same page
